plugin/storage/es/spanstore/reader.go

```diff
@@ -91,16 +95,22 @@ type SpanReader struct {
 	// The age of the oldest trace we will look for. Because indices in ElasticSearch are by day,
 	// this will be rounded down to UTC 00:00 of that day.
 	maxSpanAge time.Duration
+
+	serviceNamesReader   serviceNamesReader
```

Comment:
why the level of indirection vs calling serviceOperationStorage.getServices() directly? to mock the call?